DashboardFrontend/MainWindow.xaml.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using DashboardFrontend.DeatachedWindows;
+using DashboardFrontend.DetachedWindows;
```

Comment:
Har den begge de to her? Hvad er grunden?

Reply:
han stavede detached som "deatached"

Reply:
Nej, den skal kun bruge den ene, jeg havde ikke lige set den stadig var der.
```suggestion
using DashboardFrontend.DetachedWindows;
```

Reply:
Fino 😎

Reply:
bør de begge dog stadig være der?